CITATION.cff

```diff
@@ -32,9 +32,8 @@ authors:
     given-names: Jannes
     affiliation: "Vrije Universiteit Amsterdam"
 
-date-released: 20??-MM-DD
-doi: <insert your DOI here>
-version: "0.1.0"
+date-released: 2022-09-01
+version: "version: "0.2.0""
```

Comment:
There's a double "version" here, is that correct?

Reply:
Thanks for spotting this. Yep, this is a bug caused by `bump2version`. When I run it, it added these double versions thingy to some files...Had a discussion with Sarah but we don't know why...So I simply delete these double versions. But forgot this one. I will investigate this a bit.

Reply:
Quite strange...`bump2version` just not work...When I simply run `bump2version 0.2.0`, it keeps causing double versions and it only gives "0.1.0". It seems `bump2version` fails to recognize the version, although `.bumpversion.cfg` already exists. Also did not find anything useful after some googling. I will ask Stefen tomorrow. For now, I can only change it manually 😢 .